<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchPlayersManagerDialog</class>
 <widget class="QDialog" name="MatchPlayersManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInputComplete">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>730</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>录入完成</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetPerformance">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>331</width>
     <height>331</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>球员姓名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>球员编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>效力球队</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>三分球</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>篮板球</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>抢断球</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>扣篮</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>521</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>球队1场次表现</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxTeam1Performance">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>三分球</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>抢断球</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>篮板球</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>扣篮</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>球队2场次表现</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxTeam2Performance"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>三分球</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>抢断球</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>篮板球</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>扣篮</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBoxPerformance">
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxMatchID"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>号比赛场次球员</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBoxSelectMatchID">
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QPushButton" name="pushButtonNotRemove">
      <property name="text">
       <string>不删除球员</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>删除球队1队员</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxTeam1Players"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>删除球队2队员</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxTeam2Players"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBoxRemove">
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_12">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>178</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>            </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>新增球队1球员</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>新增球队2球员</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>球员编号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>球员姓名</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>球员年龄</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLineEdit" name="lineEditTeam1PlayerID"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTeam1PlayerName"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTeam1PlayerAge"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QLineEdit" name="lineEditTeam2PlayerID"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTeam2PlayerName"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTeam2PlayerAge"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBoxAdd">
      <property name="standardButtons">
       <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>     球员场次表现</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeam1ThreePoints">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>104</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeam1Steals">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>320</y>
     <width>104</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeam1Rebounds">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>104</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeam1Dunks">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>430</y>
     <width>104</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeam2ThreePoints">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>490</y>
     <width>104</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeam2Steals">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>550</y>
     <width>104</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeam2Rebounds">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>600</y>
     <width>104</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeam2Dunks">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>650</y>
     <width>104</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
